backend/src/main/java/com/rota/auth/AuthenticationUtils.java

```diff
@@ -12,17 +13,37 @@
    * @return Whether authentication was successful.
    */
   public static boolean validateToken(String token) {
-    return true;
+    return !token.toLowerCase().equals("false");
```

Comment:
Can you explain what's going on on this line?

Reply:
This makes it so that we have a way of providing an invalid token. 
If we pass in `false` as the token then it'll output false. Otherwise it'll be true.
(not sensitive to case)

Maybe i should add a comment there, or split it up in to separate lines to make it clearer